### PR TITLE
feat(devtools): option to specify action type

### PR DESCRIPTION
**Pull request by soyuka**
## PR Checklist

Allows to customize the action type within the dev tools.

### CODE REVIEW

File: packages/devtools/src/lib/devtools.spec.ts [hunk 1]
```diff
@@ -15,9 +15,13 @@ describe('devtools', () => {
         connect: jest.fn().mockImplementation(() => instanceMock),
       },
     },
+    configurable: true,
+    writable: true,
   });
 
-  createEntitiesStore('books');
```
Comment (soyuka) [inline packages/devtools/src/lib/devtools.spec.ts:20]: I need to clear the store or I can not add a second `describe` as the store is in an unknown state it makes the state harder to assert.

**Comment by soyuka** [comment]
> Did you check that your code works?

What code are you talking about? The stack trace hack in the documentation? It works on my project because I'm using a repository and I search for it in the stack trace. If there's another way to customize the action type I'm all ears.

**Comment by NetanelBasal** [comment]
Yes, I'm asking if you check that it works with the `getActionType` option.

**Comment by soyuka** [comment]
you're right it doesn't work anymore, I can't find any occurrence of the repository in the stack trace :/ I wonder how we could improve the developer experience though, having only `update` as action type isn't good when working with many events.

**Pull request closed without merging**